### PR TITLE
blog: merge time vs review complexity

### PR DESCRIPTION
**Pull request by GregSutcliffe**
@tbrisker hopefully this helps your Discourse discussion on review complexity :)

### CODE REVIEW

File: _posts/2019-03-04-merge-time-review-complexity.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+  - foreman
+  - data
+  - process
+  - community
+---
+
+In this post I'm going to dig into some of the data we have on GitHub about how
+our community functions. In particular, I'm looking to investigate some of the
+issues raised
+[here](https://community.theforeman.org/t/improving-pr-merge-velocity/12992) to
```
Comment (evgeni) [inline _posts/2019-03-04-merge-time-review-complexity.md]: stop. linking. to. "here". :)

`investigate some of the [issues raised on Discourse](…)`

Comment (GregSutcliffe) [reply _posts/2019-03-04-merge-time-review-complexity.md]: Sorry, bad habit. Fixed.

**Comment by GregSutcliffe** [comment]
Thanks to @tbrisker for some discussion in IRC. Updated with a reworded section on how "typical values" affect *only* the drawing of the graphs (the model uses the whole range) and changed the typical values to get a better feel for the effects in play. Much happier with the final graph now!

**Review by tbrisker — APPROVED** [review]
Thanks for the great work @GregSutcliffe !

**Pull request merged**